tests/unit/dataactbroker/test_file_handler.py

```diff
@@ -55,14 +55,14 @@ def test_list_submissions_sort_success(database, job_constants, monkeypatch):
     assert result['total'] == 5
     sub = result['submissions'][0]
     for subit in result['submissions']:
-        assert subit['user']['name'] <= sub['user']['name']
+        assert subit['user']['name'] >= sub['user']['name']
```

Comment:
Yeah they didn't seem correct. It was testing `submitted_by` in ascending order so `sub['user']['name']` in this case is `Barry Allen` which is actually less than `Oliver Queen` when doing such a comparison with strings. I'm not sure how it was working before. Strange that I was only seeing this failure now.